src/stylesheets/components/_notFound.scss

```diff
@@ -7,7 +7,26 @@
     text-align: center;
     width: 100%;
 
```

Comment:
Because of the `justify-content` property, the page contents overlap the header and do not scroll properly when the page height is small. I know this is not caused by your changes, but as part of the not-found page improvements can you please fix this?

<img width="1280" alt="screen shot 2018-07-25 at 10 26 59 pm" src="https://user-images.githubusercontent.com/9062287/43243289-c3a6ad7c-905a-11e8-80d6-0380bbc84e6b.png">

Reply:
Fixed.

Reply:
Fixed.